ols/src/tools/tools.py

```diff
@@ -53,6 +54,11 @@ def execute_oc_tool_calls(
                 tool_args["token"] = token
                 tool_output = tool.invoke(tool_args)
                 del tool_args["token"]
+            except ValidationError:
+                tool_output = (
+                    f"Error executing {tool_name}: tool arguments are in wrong format"
+                )
+                logger.exception(tool_output)
```

Comment:
shall we return the error message to the user?

Reply:
is it safe to continue calling other tools in such case? It won't answer the question, probably might run something wrong, and will just eat resources IMHO.
And the token won't be removed

Reply:
@asamal4 yup. I'd suggest to delete the token in `finally`, just to be sure it really be removed, even before you log it.